src/main/java/seedu/eduke8/storage/TopicsStorage.java

```diff
@@ -82,6 +100,12 @@ private Topic parseToTopicObject(JSONObject topic) throws Eduke8Exception {
 
     private Question parseToQuestionObject(JSONObject question) throws Eduke8Exception {
         currentQuestionDescription = (String) question.get(KEY_DESCRIPTION);
+
+        if (currentQuestionDescription.equals("")) {
+            throw new Eduke8Exception(ERROR_TOPICS_JSON_PREFACE
+                    + System.lineSeparator() + ERROR_TOPICS_JSON_DESCRIPTION_BLANK);
+        }
```

Comment:
Great that you are checking for empty titles and throwing an exception error to the user!